tests/util/kube_utils.go

```diff
@@ -275,6 +335,55 @@ func GetPodStatus(n, pod string) string {
 	return ""
 }
 
+// GetPodName gets the pod name for the given namespace and label selector
+func GetPodName(n, labelSelector string) (pod string, err error) {
+	pod, err = Shell("kubectl -n %s get pod -l %s -o jsonpath='{.items[0].metadata.name}'", n, labelSelector)
+	if err != nil {
+		log.Warnf("could not get %s pod: %v", labelSelector, err)
+		return
+	}
+	pod = strings.Trim(pod, "'")
+	log.Infof("%s pod name: %s", labelSelector, pod)
+	return
+}
+
+// GetPodLogsForLabel gets the logs for the given label selector and container
+func GetPodLogsForLabel(n, labelSelector string, container string, tail, alsoShowPreviousPodLogs bool) string {
+	pod, err := GetPodName(n, labelSelector)
```

Comment:
What about when we have more than 1 pod ? (which is soon to be the standard deployment)

Reply:
It's beyond the scope of this PR... I'm just porting over the existing tests.